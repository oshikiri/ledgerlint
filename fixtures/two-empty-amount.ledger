2020-03-26
  Expences:Household essentials  200 JPY
  Expences:Food
  Assets:Cash
