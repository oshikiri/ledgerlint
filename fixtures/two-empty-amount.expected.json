{"file_path":"fixtures/two-empty-amount.ledger","line_number":1,"error_message":"Transaction contains two or more empty amount"}
